plugins/mcreator-core/datalists/eventnumberparameters.yaml

```diff
@@ -0,0 +1 @@
+- DAMAGE_AMOUNT
```

Comment:
Datalists can have types.

Similar how we don't have separate lists for number and logic gamerules